packages/create-app/templates/default-app/docker-compose.yaml.hbs

```diff
@@ -0,0 +1,40 @@
+version: "3.4"
+services:
+  {{#if dbTypePG}}
+  # TODO: Need to figure out why databases aren't auto created
+  postgres:
+    image: bitnami/postgresql:11
```

Comment:
Why not use the official `postgres` or `postgres:alpine` images here?

Reply:
Normally I default to mirroring the production setup with Postgres, such as the case with deploying Postgres with Helm using the recommend chart. https://github.com/bitnami/charts/tree/master/bitnami/postgresql

I think it makes sense to use the slimmer alpine image since production setups will vary. Might be useful to switch back when the repo Helm chart is built out more.